src/SFA.DAS.EmployerAccounts.Web/Views/EmployerTeam/V2/VacancyStatus.cshtml

```diff
@@ -0,0 +1,35 @@
+﻿
+@using SFA.DAS.EAS.Portal.Client.Types;
+@model SFA.DAS.EmployerAccounts.Web.ViewModels.AccountDashboardViewModel
+@{
+    Vacancy vacancy = null;
+
+    var actionLinkText = "";
+    int? numberOfApplications = null;
+
+    if (Model.HasSinglePendingReviewVacancy)
+    {
+        vacancy = Model.AccountViewModel.Vacancies.First(v => v.Status == VacancyStatus.PendingReview);
+        actionLinkText = "Preview vacancy";
+    };
+}
+<h3 class="das-panel__heading">Your vacancy</h3>
+<p>@vacancy.TrainingTitle <span class="das-!-display--block das-text--muted">VAC @vacancy.Reference</span></p>
+<dl class="das-definition-list das-definition-list--inline">
+    <dt class="das-definition-list__title">Closing date</dt>
+    <dd class="das-definition-list__definition">@(vacancy.ClosingDate.HasValue ? vacancy.ClosingDate.Value.ToGdsFormat() : "-")</dd>
```

Comment:
The design shows the month in full, e.g. 'June', whereas `ToGdsFormat()` uses `MMM` for the month, which only shows the first three characters of the month.